src/OSPSuite.Core/Snapshots/SnapshotVersions.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Linq;
+using OSPSuite.Utility.Collections;
+
+namespace OSPSuite.Core.Snapshots
+{
+   /// <summary>
+   ///    Originally based on PK-Sim project versions, so the first few entries must coincide with PK-Sim project versions
+   /// </summary>
+   public static class SnapshotVersions
+   {
+      private static readonly Cache<int, SnapshotVersion> _knownVersions = new Cache<int, SnapshotVersion>(x => x.Version, x => null);
+
+      public static readonly SnapshotVersion V7_1_0 = addVersion(71, "7.1.0");
```

Comment:
I don;t think we need all of them. Snapshot were introduced later no?
Also if it's not referneced anywhere, why bother?